ricecooker/utils/add_file.py

```diff
@@ -0,0 +1,204 @@
+import requests
```

Comment:
Maybe for the module name, we should do `nodes` rather than `add_file`, so we can put any additional functions that deal with nodes in here?

Reply:
Sounds good to me.